### ISSUE TITLE
"Wrong Hanging Indentation" false positive?

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#232]
Originally reported by: **Sean Wallitsch (BitBucket: [shidarin](http://bitbucket.org/shidarin), GitHub: @shidarin?)**

---

Since 1.2.1, pylint no longer lets the following be an acceptable line continuation:

```
#!python
    parser.add_argument(
        "-o",
        "--output",
        help="specify the filetype to convert to, comma separated lists are "
             "accepted. Defaults to a .cc XML. Supported output formats are: "
             "{outputs}".format(outputs=str(OUTPUT_FORMATS.keys()))
    )
```

It's throwing C0330(bad-continuation) with the note "Wrong Hanging indentation." Seeming to prefer the following:

```
#!python
    parser.add_argument(
        "-o",
        "--output",
        help="specify the filetype to convert to, comma separated lists are "
        "accepted. Defaults to a .cc XML. Supported output formats are: "
        "{outputs}".format(outputs=str(OUTPUT_FORMATS.keys()))
    )
```

Which doesn't seem right. This looks like it was introduced in [this commit](https://bitbucket.org/logilab/pylint/commits/50f940cb47daaa8c3b87a0046bbb97cdf7b6c879), and we noticed it when the new version was being used on landscape.io (see github issue [here](https://github.com/landscapeio/landscape-issues/issues/41)). Is this a bug?

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/232
